docker-compose.yml

```diff
@@ -64,3 +64,29 @@ services:
     volumes:
     - ./frontend_v2:/code
     - /code/node_modules
+
+  prometheus:
+    image: prom/prometheus:latest
+    user: "1000"
+    volumes:
+      - ./prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+      - ./prometheus/prometheus_db:/var/lib/prometheus
+      - ./prometheus/prometheus_db:/prometheus
+      - ./prometheus/prometheus_db:/etc/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+    restart: unless-stopped
+    ports:
+      - '9090:9090'
+  
+  grafana:
+    image: grafana/grafana:latest
+    user: "1000"
+    environment:
+      - GF_SECURITY_ADMIN_PASSWORD=evalai
```

Comment:
@burnerlee move this to a `docker.env` file for grafana. We can't check in the secrets in the repository